packages/daimo-common/src/daimoLink.ts

```diff
@@ -123,7 +123,7 @@ function parseDaimoLinkInner(link: string): DaimoLink | null {
         previewSender = parts[1];
         const parsedDollars = zDollarStr.safeParse(parts[2]);
         if (!parsedDollars.success) return null;
-        previewDollars = parsedDollars.data;
+        previewDollars = parseFloat(parsedDollars.data).toFixed(2) as DollarStr;
```

Comment:
👍

Reply:
we can also fix this one the link creation side -- the link should be `/1.00` not `/1`

Reply:
probably better to keep them shorter